### ISSUE TITLE
`enable_header` should not be `true` by default

### ISSUE THREAD

**Issue by reslario** [stve/silencer#36]
This feature is questionable at best to have in production, the thought of someone not noticing it's enabled makes me a bit uneasy.

**Comment by stve** [comment]
I agree with this. I understand that some may want to have a client disable the logger but the default should be disabled. Will consider for a future release!
